### PR TITLE
 feat(presets): add 'daynight' preset

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: presets.js [hunk 1]
```diff
@@ -28,5 +28,23 @@ module.exports = {
     '\uD83D\uDD58',
     '\uD83D\uDD59',
     '\uD83D\uDD5A'
+  ],
+  daynight: [
+    '\uD83C\uDF15', // 🌕
```
Comment (rafaelrinaldi) [inline presets.js]: No need to add the comments here 👍

**Review by rafaelrinaldi — CHANGES_REQUESTED** [review]
Thank you for the PR. Just added one note. Also, can you check why the build is failing? Please, make sure tests are passing.

**Comment by marcoscaceres** [comment]
Just wondering, how come you don't include the emojis directly in the code? Is there known compat issues?

**Comment by marcoscaceres** [comment]
Not sure why the semaphoreci thing is failing. Any ideas?

**Comment by rafaelrinaldi** [comment]
@marcoscaceres Alright, I will take a look as soon as I can.

**Comment by marcoscaceres** [comment]
Friendly ping :)

**Comment by marcoscaceres** [comment]
Closing due to inactivity.

**Comment by rafaelrinaldi** [comment]
@marcoscaceres I haven't been active on GitHub much lately since I'm very busy on my full time job. I hope you understand.

I've added this to my todo list to check on the weekend. I want to avoid merging it with CI failing, so I'll figure out why it's breaking and then introduce your changes on top of it. Will `@` you on the PR.

**Comment by marcoscaceres** [comment]
@rafaelrinaldi totally understand. It's no problem. I added these mostly as a fun thing - but no stress if they don't get merged.

**Pull request closed without merging**